packages/debug/src/browser/console/debug-console-items.tsx

```diff
@@ -103,6 +103,10 @@ export class ExpressionContainer implements CompositeConsoleItem {
             const response = await this.session!.sendRequest('variables', { variablesReference, filter, start, count });
             const { variables } = response.body;
             const names = new Set<string>();
+            for (const item of result) {
+                const variable = item as DebugVariable
```

Comment:
Directly casting from `ConsoleItem` to `DebugVariable` seems a bit unsafe to me. Since `DebugVariable` is a class, you should be able to use the `item instanceof DebugVariable` check to assert that it definitely is a `DebugVariable`. Alternatively, you could first assert that `item` contains a `name` property and then cast to something like `{ name: string }`.